$MirOS: ports/multimedia/mplayer/patches/patch-libvo_font_load_ft_c,v 1.1.7.1 2005/03/18 15:50:49 tg Exp $
--- libvo/font_load_ft.c.orig	Thu Oct 28 01:15:30 2004
+++ libvo/font_load_ft.c	Thu May 19 20:32:10 2005
@@ -917,7 +917,7 @@ static int load_sub_face(char *name, FT_
 	if (err) {
 	    err = FT_New_Face(library, MPLAYER_DATADIR "/subfont.ttf", 0, face);
 	    if (err) {
-	        mp_msg(MSGT_OSD, MSGL_ERR, "New_Face failed. Maybe the font path is wrong.\nPlease supply the text font file (~/.mplayer/subfont.ttf).\n" );
+	        mp_msg(MSGT_OSD, MSGL_ERR, "New_Face failed. Maybe the font path is wrong.\nPlease supply the text font file (~/.etc/mplayer/subfont.ttf).\n" );
 		return -1;
 	    }
 	}
